pontoon/base/models.py

```diff
@@ -76,6 +76,10 @@ def cldr_plurals_list(self):
         else:
             return map(int, self.cldr_plurals.split(','))
 
+    @property
+    def dir(self):
```

Comment:
I agree and I thought about that. But with Locales being static, I wonder if we should completely remove them from the DB? Shouldn't that also have (positive) impact on performance?